materialsLibrary/src/shadowOnly/babylon.shadowOnlyMaterial.ts

```diff
@@ -107,12 +107,29 @@ module BABYLON {
                                 "vClipPlane", "depthValues"
                 ];
                 var samplers = [];
-                    
-                MaterialHelper.PrepareUniformsAndSamplersList(uniforms, samplers, defines, 1);
+                
+                var uniformBuffers = [];
+
+                MaterialHelper.PrepareUniformsAndSamplersList(<EffectCreationOptions>{
+                    uniformsNames: uniforms, 
+                    uniformBuffersNames: uniformBuffers,

```

Comment:
`PrepareUniformsAndSamplersList` could create it if undefined. I would like to reduce the code required to use it